deploy/charts/rook-ceph/values.yaml

```diff
@@ -253,6 +261,33 @@ csi:
   #      limits:
   #        memory: 256Mi
   #        cpu: 100m
+  # (Optional) CEPH CSI CephNFS plugin resource requirement list, Put here list of resource
+  # requests and limits you want to apply for plugin pod
+  # csiCephNFSPluginResource: |
+  #  - name : driver-registrar
```

Comment:
With #9894 merged now, let's uncomment these resource limits/requests in the helm chart.